GitCommands/Git/RefName.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using JetBrains.Annotations;
+
+namespace GitCommands
+{
+    public static class RefName
```

Comment:
For consistency I suggest we call it `GitRefName` or maybe even `GitRef`, since this class does not represent ref name, but provides capability to get it.

Reply:
`GitRef` already exists, so went with `GitRefName`.

Reply:
Aye, didn't see it. 👍